resource-groups/rds-postgres/main.tf

```diff
@@ -5,6 +5,7 @@ resource "aws_db_subnet_group" "subnet_group" {
 
 resource "aws_db_instance" "db" {
   allocated_storage               = var.allocated_storage_gb
+  auto_minor_version_upgrade      = false
```

Comment:
Because this is in the core repo it will affect the other ~6 or so systems under migration, maybe better to make this a `var`, default it to `true` then pass in the `false` for DMP specifically.

Also perhaps worth checking in with @ghill95 to see how he feels about this change across the board. (George for conetxt this is to prevent AWS performing upgrades and breaking version matching between GPaaS and RDS before we have migrated the db via a dump [it's advised that both db versions are the same for this process])

Reply:
Discussed with @ghill95 on the standup call this morning; he is of the opinion that CCS will want to manage any and all DB version upgrades themselves and that AWS should not be allowed to auto-upgrade across any of their environments.